### PR TITLE
tests: Add gradle and nextcloud tests

### LINKED ISSUE #1921: Add gradle and nextcloud images tests

### PR DESCRIPTION
**Pull request by GabyCT**
This adds the gradle and nextcloud docker hub images tests to the
popular docker images.

Fixes #1921

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/popular_docker_hub_images/popular_docker_images.bats [hunk 1]
```diff
@@ -161,6 +161,12 @@ setup() {
 	docker run --rm --runtime=$RUNTIME -i $image bash -c "echo -e '#include<stdio.h>\nint main (void)\n{printf(\"Hello\");return 0;}' > demo.c && gcc demo.c -o demo && ./demo"
 }
 
+
+@test "[gradle] run a gradle container" {
+	image="gradle"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "echo -e 'public class CL{public static void main(String[]a){System.out.println(\"KataContainers\");}}' > CL.java"
```
Comment (ganeshmaharaj) [inline integration/popular_docker_hub_images/popular_docker_images.bats]: Shouldn't we try to compile this or is the intent of the test just to write the file and exit?

Comment (GabyCT) [reply integration/popular_docker_hub_images/popular_docker_images.bats]: @ganeshmaharaj changes applied

**Comment by eadamsintel** [comment]
I've been running a Nextcloud Kata container on my personal server for over a month now using the default overlay storage. I've not noticed any issues yet but I only am sharing a few folders with friends. The Kata on clear linux on my server though is still stuck at 1.6.2

**Pull request merged**